recipe/conda_build_config.yaml

```diff
@@ -11,28 +11,29 @@ vsver:
  - 16
  - 15
 runtime_version:
- - 14.29.30037
```

Comment:
Can you add a comment (or just a URL to the issue) here about why we need different minor versions?